assets/xml/objects/object_zl4.xml

```diff
@@ -1,125 +1,177 @@
+        <Animation Name="gDmZl4TurningAround2Anim" Offset="0x15494" />
+        <Animation Name="gDmZl4HandsOverEmblemLoopAnim" Offset="0x15A4C" />
+        <Animation Name="gDmZl4FacingAwayHandsOverEmblemLoop" Offset="0x15DA0" />
+
+        <!-- Unused: most of these look like they could be from OoT --> 
+        <!-- this one is weird... -->
+        <Animation Name="gDmZl4Anim_016328" Offset="0x16328" />
+        <Animation Name="gDmZl4Anim_016924" Offset="0x16924" />
+        <!-- washing hands is important? no idea what this is -->
+        <Animation Name="gDmZl4Anim_016E04" Offset="0x16E04" />
+        <Animation Name="gDmZl4Anim_0175F4" Offset="0x175F4" />
+        <!-- caught peeking at ganondorf through the window? -->
+        <Animation Name="gDmZl4Anim_018B58" Offset="0x18B58" />
+        <Animation Name="gDmZl4Anim_018FA8" Offset="0x18FA8" />
+        <Animation Name="gDmZl4Anim_0193F8" Offset="0x193F8" />
+        <Animation Name="gDmZl4Anim_019894" Offset="0x19894" />
+        <Animation Name="gDmZl4Anim_01A344" Offset="0x1A344" />
+        <!-- writing the letter for OoT progression -->
+        <Animation Name="gDmZl4Anim_01ABB8" Offset="0x1ABB8" />
+        <Animation Name="gDmZl4Anim_01AFE0" Offset="0x1AFE0" />
+        <!-- head tilting while talking to you about her dreams of ganondorf? -->
+        <Animation Name="gDmZl4Anim_01B388" Offset="0x1B388" />
+        <Animation Name="gDmZl4Anim_01B7B0" Offset="0x1B7B0" />
+        <Animation Name="gDmZl4Anim_01BC00" Offset="0x1BC00" />
+        <Animation Name="gDmZl4Anim_01DC74" Offset="0x1DC74" />
+        <Animation Name="gDmZl4Anim_01E714" Offset="0x1E714" />
+        <Animation Name="gDmZl4Anim_01EA68" Offset="0x1EA68" />
+        <Animation Name="gDmZl4Anim_01EFCC" Offset="0x1EFCC" />
+        <!-- do you understand what I said, head tilt? -->
+        <Animation Name="gDmZl4Anim_01F578" Offset="0x1F578" />
+        <Animation Name="gDmZl4Anim_01FDBC" Offset="0x1FDBC" />
+        <Animation Name="gDmZl4Anim_0205F8" Offset="0x205F8" />
+        <Animation Name="gDmZl4Anim_020E6C" Offset="0x20E6C" />
+        <Animation Name="gDmZl4Anim_021360" Offset="0x21360" />
+        <Animation Name="gDmZl4Anim_02167C" Offset="0x2167C" />
+        <!-- turning to face her side to talk to link next to window? -->
+        <Animation Name="gDmZl4Anim_02205C" Offset="0x2205C" />
+        <Animation Name="gDmZl4Anim_022840" Offset="0x22840" />
+        <Animation Name="gDmZl4Anim_022BE8" Offset="0x22BE8" />
+        <!-- showing off something to her side? -->
+        <Animation Name="gDmZl4Anim_0235D4" Offset="0x235D4" />
+        <Animation Name="gDmZl4Anim_023A50" Offset="0x23A50" />
+        <!-- shock? you chose no at the wrong time? -->
+        <Animation Name="gDmZl4Anim_0241F4" Offset="0x241F4" />
+        <Animation Name="gDmZl4Anim_024510" Offset="0x24510" />
+        <!-- arms behind back head down -->
+        <Animation Name="gDmZl4Anim_024B68" Offset="0x24B68" />
```

Comment:
These are identical to OoT (of course the pointers are different since the object is shifted). Essentially this object is 
 - identical to OoT down to 0xE050
 - new animations
 - 0x15DB0–0x24B80 = OoT's 0xE050–end
 - New flute DLs

and this is IMO worth noting in comments (this object does not change at all between versions in OoT).

You'll find that most, if not all, are documented in OoT's z_en_zl4.c: https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_En_Zl4/z_en_zl4.c#L132-L177 which you can refer to for preliminary documentation if you want; all you have to do is shift the addresses. Otherwise there's not a lot of point in documenting these in a less useful way, we may as well wait for the OoT names.